Vagrant/Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# Use as a strating point to spin up a box in lightsail.
```

Comment:
Need a Vagrant run in the README, set that up real quick

Reply:
Can I leave this to you? I'm just putting this as-is, I do not intend to make it useful nor to explain my setup. 
That what I used as a dev environment, I'm just putting this in case somebody would be interested.

Reply:
I have no idea how vagrant works, so we'll just add an issue about it I guess